### PR TITLE
FEM-2272 fix warnings

### PR DESCRIPTION
**Pull request by giladna**

### CODE REVIEW

File: googlecast/src/main/java/com/kaltura/playkit/plugins/googlecast/caf/adsconfig/VastAdsConfig.java [hunk 1]
```diff
@@ -32,10 +32,7 @@
 
     @Override
     public boolean isAdsConfigValid() {
-        if (getVastAdBreakClipInfoList() == null || getVastAdBreakInfoList() == null) {
-            return false;
-        }
-        return true;
+        return !(getVastAdBreakClipInfoList() == null || getVastAdBreakInfoList() == null);
```
Comment (noamtamim) [inline googlecast/src/main/java/com/kaltura/playkit/plugins/googlecast/caf/adsconfig/VastAdsConfig.java]: De morganize it :-)

File: googlecast/src/main/java/com/kaltura/playkit/plugins/googlecast/caf/adsconfig/VmapAdsConfig.java [hunk 2]
```diff
@@ -26,10 +26,7 @@
 
     @Override
     public boolean isAdsConfigValid() {
-        if (getVmapAdRequest() == null || (getVmapAdRequest().getVastAdsRequestForAdTag() == null && getVmapAdRequest().getVastAdRequestForAdResponse() == null)) {
-           return false;
-        }
-        return true;
+        return !(getVmapAdRequest() == null || (getVmapAdRequest().getVastAdsRequestForAdTag() == null && getVmapAdRequest().getVastAdRequestForAdResponse() == null));
```
Comment (noamtamim) [inline googlecast/src/main/java/com/kaltura/playkit/plugins/googlecast/caf/adsconfig/VmapAdsConfig.java]: De morganize it :-)

**Review by noamtamim — CHANGES_REQUESTED** [review]
De morganize!

File: googlecast/src/main/java/com/kaltura/playkit/plugins/googlecast/caf/adsconfig/VastAdsConfig.java (hunk 1 above)
Comment (giladna) [reply googlecast/src/main/java/com/kaltura/playkit/plugins/googlecast/caf/adsconfig/VastAdsConfig.java]: done

**Review by noamtamim — APPROVED** [review]

**Pull request merged**